src/QA/SoftMocks.php

```diff
@@ -94,6 +94,15 @@ protected function p(\PhpParser\Node $node)
         return $prefix . $this->{'p' . $node->getType()}($node);
     }
 
+    public function __call($name, $args)
```

Comment:
Yeah, I expected that you have your own vision on your code.
I was thinking whether to file bugreport or pull-request.
I decided that pull-request would be more polite way)
Anyway, I removed __call from this pull-request, it's now only 2-lines diff.